docker-compose.link.yaml

```diff
@@ -0,0 +1,4 @@
+networks:
+  default:
+    external:
+      name: ffc-pay

```

Comment:
does this need to be ffc-doc?

Reply:
All of the other services have kept the ffc-pay network so I left this as-is